packages/dappmanager/src/modules/chains/drivers/index.ts

```diff
@@ -25,8 +25,8 @@ export async function runWithChainDriver(
     case "ethereum":
       return await ethereum(dnp);
 
-    case "ethereum2-beacon-chain-prysm":
-      return await ethereum2Prysm(dnp);
+    case "ethereum2" || "ethereum2-beacon-chain-prysm":
```

Comment:
Is this valid syntax? I think here you are testing only `"ethereum2"``can you unit test a case for `"ethereum2-beacon-chain-prysm"`?

Reply:
fixed or